mrp_production_split/wizards/mrp_production_split_wizard.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2023 Camptocamp SA (https://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from typing import List, Union
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class MrpProductionSplitWizard(models.TransientModel):
+    _name = "mrp.production.split.wizard"
+
+    production_id = fields.Many2one(
```

Comment:
readonly?

Reply:
Not needed in fact, the field is not shown. And making it readonly would probably require `force_save="1"`
But also doesn't make much sense for a wizard, IMO.